platform/zilliqa/model.go

```diff
@@ -12,3 +21,70 @@ type Tx struct {
 	Nonce          uint64 `json:"nonce"`
 	ReceiptSuccess bool   `json:"receiptSuccess"`
 }
+
+type ChainInfo struct {
+	NumTxBlocks   string `json:"NumTxBlocks"`
+}
+
+type TxRPC struct {
+	ID       string `json:"ID"`
+	Amount   string `json:"amount"`
+	GasLimit string `json:"gasLimit"`
+	GasPrice string `json:"gasPrice"`
+	Nonce    string `json:"nonce"`
+	Receipt  struct {
+		CumulativeGas string `json:"cumulative_gas"`
+		EpochNum      string `json:"epoch_num"`
```

Comment:
can you create a separate struct for `Receipt` ?

Reply:
This `Receipt` is used only here, I would rather keep it nested